assets/lightbulb_icon.svg

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="13px" height="18px" viewBox="0 0 13 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" jetway-hooks="{}">
+    <!-- Generator: Sketch 62 (91390) - https://sketch.com -->
```

Comment:
Make sure this asset goes through an SVG optimizer, e.g. https://jakearchibald.github.io/svgomg/

This comment should be removed for example.

Reply:
Optimized